src/main/java/com/github/sider/javasee/command/CheckCommand.java

```diff
@@ -47,30 +47,33 @@ public String getName() {
         formatter.onStart();
 
         try {
-            if(!configPath().isFile()) {
+            if(!configPath().isFile() || !configPath().exists()) {
                 err.println("Configuration file " + configPath() + " does not look a file.");
                 err.println("Specify configuration file by -config option.");
-                return JavaSee.ExitStatus.CONFIG_FILE_UNKNOWN_ERROR;
+                return JavaSee.ExitStatus.CONFIG_FILE_NOT_FOUND;
             }
             var rootPath = Optional.ofNullable(optionRoot).map((root) -> new File(root)).orElse(configPath().getParentFile());
             Map<String, Object> yaml;
             Config config = null;
             try {
                 yaml = new Yaml().load(new FileInputStream(configPath()));
                 if(yaml == null) {
-                    System.out.println("YAML file has unknown error");
+                    System.out.println("YAML file \"" + configPath() + "\" has unknown error");
```

Comment:
[suggestion] Can we output the all error messages to STDERR like this?

```suggestion
                    err.println("YAML file \"" + configPath() + "\" has unknown error");
```

Reply:
I've fixed them. @ybiquitous

Reply:
Thanks!